cypress/integration/delete-collection.ts

```diff
@@ -0,0 +1,24 @@
+describe("Let's delete from collection", () => {
+  it('Adds cards from search page', () => {
+    cy.visit('/')
+    cy.contains('Search').click()
+    cy.wait(2000)
+    const cardButtons = cy.get('figure > figcaption > div').find('>button')
+
+    cardButtons.each((cardButton) => {
+      cy.wrap(cardButton).click({ force: true })
```

Comment:
I would force: true here either since the button isn't really visible, I'd hover first then click on it without the force: true